packages/loader/driver-utils/src/test/blobAggregation.spec.ts

```diff
@@ -172,15 +172,15 @@ describe("BlobAggregationStorage", () => {
         const { storage } = await prep(true, 2048);
 
         // Number of actual blobs in storage should be 2!
-        assert(storage.blobs.size === 3,
```

Comment:
This does not seem right. The `summaryTree` used in the test should be updated to reflect the new format with ".channels".